### DISCUSSION TITLE
any way to stop infinite recursion?

### DISCUSSION THREAD

**Discussion by ghost** [beakerbrowser/beaker#1832] [Q&A]
like, lets say i linked `.ui/` to root of the drive,
since it's a drive that holds specific `.ui`
seems logical, right?

but somehow javascript does not have a concept of lazy evaluation,
and tries to infinitely index that recursive folder

**Comment by pfrazee** [comment]
The way I've handled that historically is just have my JS detect by looking at the `location.pathname` and aborting if it's `/.ui/index.html`

**Comment by ghost** [comment]
follow-up question: moving link to another folder, or file which has link to it, causes another infinite loop
